### ISSUE TITLE
Wont connect to snapchat during install.

### ISSUE THREAD

**Issue by TGEJesse** [sallaben/storybot#5]
I'm 100% sure my information is correct and I get this:

Incorrect snapchat username!
Please press 'back' on your browser and try again.

No matter what I do during the install.

**Comment by sallaben** [comment]
yeah man this code is pretty outdated. how much php do you know?

it could be as simple as replacing the /src/ folder with the new files from mgp25's "SC-API" repo, though.

**Comment by TGEJesse** [comment]
I was able to fix that but then I think I got rate limited... I get this when I run the test file:

stdClass Object
(
    [message] => The network you are connected to has been temporarily blocked because of suspicious activity. If you are connected to a VPN, retry after disconnecting from it.
    [status] => 403
    [logged] => 
)
RESULT: 1

**Comment by sallaben** [comment]
yep that'll rate limit you. one thing I might suggest is to keep sessions -
instead of logging in every time you make a request, have the program use
the same session for multiple requests. Which is why PHP probably isn't the
best for this type of API.
